OpenRA.Mods.Common/Traits/RegionProximityCapturable.cs

```diff
@@ -0,0 +1,71 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Graphics;
+using OpenRA.Primitives;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Actor can be captured by units entering a certain set of cells.")]
+	public class RegionProximityCapturableInfo : ProximityCapturableBaseInfo
+	{
+		[Desc("Set of cell offsets (relative to the actor's Location) the ProximityCaptor needs to be in to initiate the capture. ",
+			"A 'Region' ActorInit can be used to override this value per actor. If either is empty or non-existent, ",
+			"the immediately neighboring cells of the actor will be used.")]
```

Comment:
Consider applying `nameof`.

Reply:
Done, except for the ActorInit. It would say "RegionInit" and I suppose that would confuse modders (ie. they'd try adding "RegionInit" to the yaml instead of just "Region".